mackerel-plugin-gcp-compute-engine/README.md

```diff
@@ -7,23 +7,25 @@ mackerel-plugin-gcp-compute-engine is mackerel plugin that collects metrics from
 
 ## Description
 
-This plugin uses Google Monitoring API to get more metrics than mackerel-agent defalut metrics.   
+This plugin uses Google Monitoring API to get more metrics than mackerel-agent defalut metrics.
 
 
-Caution: This Plugin works only on Compute Engien instance that is enabled Stackdriver Monitoring API full access. 
+Caution: This Plugin works only on Compute Engien instance that is enabled Stackdriver Monitoring API full access.
 
 ## Usage
 
 ```shell
-mackerel-plugin-gcp-compute-engine [-api-key=<APIKey>]
+mackerel-plugin-gcp-compute-engine -api-key=<APIKey> [-project=<project_id_or_number>] [-instance-name=<target instance name>]
```

Comment:
[nit] The format (e.g. capitalization) of `APIKey` , `project_id_or_number` , and `target instance name` (and `YOUR-API-KEY` below this) seem inconsistent.